.gitmodules

```diff
@@ -3,8 +3,8 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/nitro-contracts"]
 	path = lib/nitro-contracts
-  url = https://github.com/EspressoSystems/nitro-contracts
-  branch = develop
+	url = https://github.com/EspressoSystems/nitro-contracts
+	branch = develop
```

Comment:
the develop branch has v3.1.0 contracts, orbit actions doesnt support v3.1.0 right now